### DISCUSSION TITLE
Thruk demo - Thruk looks awfull

### DISCUSSION THREAD

**Discussion by tomaszdubiel18** [sni/thruk#1191] [General]
Is this what Thruk is supposed to look like?
Tested on Firefox and Chrome.
![obraz](https://user-images.githubusercontent.com/85760200/184926850-ea10b37b-9e3b-4c07-8930-f6d65dfbbc0d.png)

**Comment by sni** [comment]
no, this is how it looks if no css could be loaded. Which version is that?

**Reply by tomaszdubiel18** [reply]
This is Thruk demo opened in Chrome and Firefox. Previously it was ok. I dont know if this is connected with new Thruk.
I will check soon with Edge.

**Reply by tomaszdubiel18** [reply]
On Edge the same
![obraz](https://user-images.githubusercontent.com/85760200/184928107-619c265d-705d-4f0f-8813-71ea208c2bd7.png)

**Comment by sni** [comment]
should be fine again. The demo.thruk updater failed to recreate the stylesheets. It's working again.

**Reply by tomaszdubiel18** [reply]
Yeah, now it's ok. For the moment I was scared with this new Thruk look :-D
